homeassistant/helpers/aiohttp_client.py

```diff
@@ -59,6 +58,17 @@
 MAXIMUM_CONNECTIONS_PER_HOST = 100
 
 
+# Overwrite base aiohttp _wait implementation
+# Homeassistant has a custom shutdown wait logic.
```

Comment:
@cdce8p Would a cleaner approach work by just setting shutdown_timeout to 0?

If not, then maybe there are some improvements to be made in aiohttp...

I'd also be curious how homeassistant's shutdown logic works.